### ISSUE TITLE
Allow running upgrades when Plone site is outdated

### ISSUE THREAD

**Issue by mauritsvanrees** [4teamwork/ftw.upgrade#182]
Summary: Currently ftw.upgrade refuses to run an upgrade step when the Plone Site is outdated. Can we get an `--allow-outdated` or similar option to allow running a step anyway?
I might be able to make a PR for this.

Background:

I have an old Plone 4.3.7 site that I am upgrading to 4.3.19.
It has a problem dating far back. The toolset has a list of required tools, and one of them cannot be created. This is not a problem until some unrelated package has a `toolset.xml` that gets applied. Plone then tries to create the tool, and this fails.
Problem: this happens during the Plone upgrade. One of the many upgrade steps applies `toolset.xml` from `plone.app.discussion`. This then fails.

I have a way around this: I wrote an upgrade step in a policy package that applies a fixed `toolset.xml`. The package already had a fine `toolset.xml`, but that did not get properly handled until I fixed something years ago in Plone 4.3.8 or 4.3.9. (I wonder if I did that for this customer, but never had a chance to apply it...). The upgrade step works.

Now I must apply this upgrade _before_ I do the Plone upgrade.
Problem: `ftw.upgrade` does not allow this:

```
$ bin/upgrade install --last-site --upgrades 5@zindex.website:default
ERROR: Plone site outdated: The Plone site is outdated and needs to be upgraded first using the regular Plone upgrading tools.
> http://localhost:8080/z-index/upgrades-api/execute_upgrades?upgrades%3Alist=5%40zindex.website%3Adefault
< 400 Plone site outdated
```

So there is a catch-22:

- I must apply my upgrade step first, otherwise the Plone upgrade fails.
- But I cannot apply my upgrade step, because I first have to upgrade Plone.

The obvious workaround is to do my upgrade step manually.

But can we have an option on the command line to allow running upgrades for an outdated site?

The same could be useful for the `--profiles` option.

I guess this would not be good for the `--proposed` option, because that would probably do the Plone upgrade itself, and not necessarily in the correct order.

**Comment by jone** [comment]
Sure, I think that a `--allow-outdated`-option would be a good solution 👍 
Do you want to make a pull request?

**Comment by mauritsvanrees** [comment]
This was fixed a day later with the 2.14.0 release. Closing.
